datafusion/core/tests/tpcds_planning.rs

```diff
@@ -343,7 +343,6 @@ async fn tpcds_logical_q63() -> Result<()> {
     create_logical_plan(63).await
 }
 
-#[ignore] // thread 'q64' has overflowed its stack]
```

Comment:
I think these tests will likely still fail due to https://github.com/apache/arrow-datafusion/issues/4786 but I figured I would let the CI run and see if we can squeek by

Reply:
They still fail: https://github.com/apache/arrow-datafusion/actions/runs/3817136119/jobs/6493197591